manifests/cf-manifest/deployments/030-cf-jobs.yml

```diff
@@ -1,6 +1,9 @@
 meta:
   environment: ~
 
+  api_domain: (( concat "api." properties.domain ))
+  apps_domain: (( concat "apps." properties.domain ))
```

Comment:
I agree, it's important, which means that we shouldn't slip it in with the more innocuous smoke test work. Please rebase it out and ask Rory/Brett to raise a separate story. You can push it to a separate branch and reference that in the story so that we don't lose it.